viewdir/index.njk

```diff
@@ -1,6 +1,132 @@
+{% extends "layout-template.njk" %}
+
+{%block pageTitle%}
+    Login
+{% endblock %}
+
 {% block content%}
-<h1>{{ email }}</h1>
-<form action='/logout' method="GET">
-    <input type="submit" value="Logout" />
-</form>
-{% endblock%}
\ No newline at end of file
+    <head>
+        <title>Kainos - Home</title>
+    </head>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <body>
+        <div class="container" style="margin-top: 2%">
+            <div class="row">
+                <div class="col s4">
+                    <a href="http://localhost:6555/JobsTable">
+                        <div class="card-panel hoverable" style="background-color: #61A83F ; margin-top: 10%">
+                            <div class="row white-text" style="margin: auto">
+                                <div class="col s4">
+                                <i class="material-icons" style="font-size: 50px">work</i>
+                                </div>
+                                <div class="col s8">
+                                    <p>Jobs</p>
+                                </div>
+                            </div>
+
+                        </div>
+                    </a>
+                    <div class="card-panel hoverable" style="background-color: #61A83F ; margin-top: 10%">
+                        <div class="row white-text" style="margin: auto">
+                            <div class="col s4">
+                                <i class="material-icons" style="font-size: 50px ; ">filter_list</i>
```

Comment:
Why is the text shown with an underscore? Is it trendy?

Reply:
This is the name of the material design icons. The text for the card is found in the <p> tag below.